### ISSUE TITLE
Seems to be broken - does not work anymore over CDN

### ISSUE THREAD

**Issue by ThomasHornschuh** [shanewholloway/js-u8-mqtt#22]
Since today u8-mqtt delivered over the CDN is not working anymore.

Even your own Demo Page: [https://shanewholloway.github.io/js-u8-mqtt/demo/simple/web-cdn.html](url)

does not work with Chrome (120.0.6099.225 ) - not tested with other Browsers yet. 
It fails with the following Stacktrace: 

`````
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at klass.with (core.jsy:30:8)
    at new MQTTCore (core.jsy:21:5)
    at new <anonymous> (with_topic_router.jsy:12:3)
    at mqtt_v4 (v5.js:15:3)
    at web-cdn.html:65:17
````

**Comment by shanewholloway** [comment]
Apologies for the breakage. Please consider using the point revision that worked last for you until I can resolve

**Comment by ThomasHornschuh** [comment]
Version 0.5.3 works...

**Comment by shanewholloway** [comment]
Found and fixed in v0.6.1. Bug affected default invocations of mqtt clients with no options specified.

**Comment by shanewholloway** [comment]
Verified working from the CDN.
